script/ci-release-build.js

```diff
@@ -136,11 +134,9 @@ function buildCircleCI (targetBranch, options) {
 }
 
 async function buildVSTS (targetBranch, options) {
-  // FIXME(codebytere): re-enable
-  // if (options.armTest) {
-  //   assert(vstsArmJobs.includes(options.job), `Unknown VSTS CI arm test job name: ${options.job}. Valid values are: ${vstsArmJobs}.`)
-  // }
-  if (options.job) {
+  if (options.armTest) {
+    assert(vstsArmJobs.includes(options.job), `Unknown VSTS CI arm test job name: ${options.job}. Valid values are: ${vstsArmJobs}.`)
+  } else if (options.job) {
```

Comment:
This is a slight change in behavior from the commented out code

Reply:
Huh?  I just reverted #16479

Reply:
options.armTest and options.job are mutually exclusive